aptos-move/block-executor/src/view.rs

```diff
@@ -229,6 +257,30 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         }
         ret
     }
+
+    /// Given a state value, performs deserialization-serialization round-trip
+    /// to replace any aggregator / aggregator snapshot values.
+    fn lift_aggregators_v2_from_state_value(
+        &self,
+        state_value: StateValue,
+        layout: &MoveTypeLayout,
+    ) -> anyhow::Result<StateValue> {
+        state_value.map_bytes(|bytes| {
+            let patched_value =
+                deserialize_and_exchange(bytes.as_ref(), layout, self).ok_or_else(|| {
```

Comment:
The word exchange is a little bit confusing. It could either id --> value or value --> id. It would be nice to use a more explicit name.

Reply:
Maybe `deserialize_and_replace_with_id`?

Reply:
If we want to keep exchange generic, we need the old name though. I added a detailed comment there